lib/perl/Genome/Sys/Lock.pm

```diff
@@ -151,30 +159,63 @@ sub with_default_lock_resource_args {
 
     $args{block_sleep} = 60 unless defined $args{block_sleep};
     $args{max_try} = 7200 unless defined $args{max_try};
+    $args{scope} = $ENV{GENOME_SYS_LOCK_DEFAULT_SCOPE} unless defined $args{scope};
```

Comment:
I've got myself worried again.  You should try running the ClinSeq model tests on this before merging; `/gsc/scripts/opt/genome/bin/model-test 5.10-clinseq-wer`.  Maybe, we should figure out a way to test the circular dependency in Perl tests so that's not necessary.

I'm thinking that have the `default_value` will be carried forward to wherever ClinSeq calls `illumina_info`, or whatever, which will then call into LIMS's Genome snapshot which won't recognize the variable and will crash.